dispatch/tests/test_api_articles.py

```diff
@@ -61,6 +61,20 @@ def test_create_article(self):
         self.assertEqual(response.data['authors'][0]['full_name'], 'Test Person')
         self.assertEqual(response.data['slug'], 'test-article')
 
+    def test_check_slug(self):
+        """Ensure that article doesn't have slug matching existing article"""
+
```

Comment:
Ah my bad -- I should have been more clear earlier!

The code below is good, but instead of replacing the old test, we actually need to create two cases:
1. Creating a **new** article with the slug of on existing article should fail
2. Updating an **existing** article with the slug of an existing article should fail

It might make sense to write these as two separate test methods:

```py
# Case 1
def test_create_article_existing_slug(self):
  ...

# Case 2
def test_update_article_existing_slug(self):
  ...
```

Case 2 is already implemented below, so all you'll have to do is add the first one back.